src/hooks/usePatchStatusSelect.ts

```diff
@@ -41,14 +47,16 @@ const reducer = (state: State, action: Action) => {
   }
 };
 
-type FilterSetter = (statuses: string[]) => void;
+export type FilterSetter = (statuses: verionFilters) => void;
```

Comment:
Does this need to be exported i dont see it imported anywhere else. I may have missed it tho.